[2021-05-25 21:33:12,972] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:30:30.633240+00:00 [queued]>
[2021-05-25 21:33:13,064] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:30:30.633240+00:00 [queued]>
[2021-05-25 21:33:13,065] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:33:13,066] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-25 21:33:13,066] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:33:13,131] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T21:30:30.633240+00:00
[2021-05-25 21:33:13,152] {standard_task_runner.py:52} INFO - Started process 423 to run task
[2021-05-25 21:33:13,171] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T21:30:30.633240+00:00', '--job-id', '136', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmp6gev9kql', '--error-file', '/tmp/tmpwk9dnmev']
[2021-05-25 21:33:13,183] {standard_task_runner.py:77} INFO - Job 136: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 21:33:13,452] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:30:30.633240+00:00 [running]> on host 4e0af5d83701
[2021-05-25 21:33:13,901] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T21:30:30.633240+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-25T21:30:30.633240+00:00
[2021-05-25 21:38:08,989] {local_task_job.py:77} ERROR - Received SIGTERM. Terminating subprocesses
[2021-05-25 21:38:09,213] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 423
[2021-05-25 21:38:09,214] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
